managed/ui/src/actions/customers.js

```diff
@@ -13,6 +13,10 @@ export const VALIDATE_FROM_TOKEN_RESPONSE = 'VALIDATE_FROM_TOKEN_RESPONSE';
 export const REGISTER = 'REGISTER';
 export const REGISTER_RESPONSE = 'REGISTER_RESPONSE';
 
+// Validate Customer registration
+export const VALIDATE = 'VALIDATE';
+export const VALIDATE_RESPONSE = 'VALIDATE_RESPONSE';
```

Comment:
Can you also change these action types to reflect what you are doing: `FETCH_PASSWORD_POLICY` seems like a better choice of wording.